pkg/resource/feature_group/testdata/feature_group/cr/fg_cmd_invalid_before_create.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
```

Comment:
would it be possible to (eventually) write scripts to generate these files during the unit test runs based on the latest SDK ?

Reply:
I think it might be possible to write a script to generate a single lifecycle of a resource, similar to what exists in PR #68 (The Feature Group state transition unit tests), based off a base file (which for the feature group state transitions, the base file was the integration test resource feature group yaml file). I essentially did the process by hand, but the unit testing state transition process could probably be replicated in code.

If you then ran the script with multiple base files, you could get many state transition variations depending on the base file, and maybe some optional input about how the sagemaker json mocks should respond. Look at Option number 2 in the sagemaker-controller unit testing design doc to get some other ideas as well on how to allow unit testing without a programmer needing to make and keep track of as many files.